plugin.xml

```diff
@@ -1,10 +1,10 @@
 <?xml version="1.0" encoding="UTF-8" standalone="yes"?>
-<plugin xmlns="http://www.phonegap.com/ns/plugins/1.0" xmlns:android="http://schemas.android.com/apk/res/android" xmlns:rim="http://www.blackberry.com/ns/widgets" id="phonegap-plugin-barcodescanner" version="6.0.3">
+<plugin xmlns="http://www.phonegap.com/ns/plugins/1.0" xmlns:android="http://schemas.android.com/apk/res/android" xmlns:rim="http://www.blackberry.com/ns/widgets" id="phonegap-plugin-barcodescanner-ilost" version="6.0.3">
   <name>BarcodeScanner</name>
   <description>You can use the BarcodeScanner plugin to scan different types of barcodes (using the device's camera) and get the metadata encoded in them for processing within your application.</description>
   <license>MIT</license>
-  <repo>https://github.com/phonegap/phonegap-plugin-barcodescanner</repo>
-  <issue>https://github.com/phonegap/phonegap-plugin-barcodescanner/issues</issue>
+  <repo>https://github.com/ilostnl/phonegap-plugin-barcodescanner</repo>
+  <issue>https://github.com/ilostnl/phonegap-plugin-barcodescanner/issues</issue>
```

Comment:
My apologies!
Can this be reverted?
This is my first fork from a public that lives in the public domain as well..